File: firstcode/firstassignemt/newassignement/digitseperator.py
def digitseperator(number): 

	first_division = (number // 10);  
	first_index = (number % 10); 

	second_division = (first_division // 10); 
	second_index = (first_division % 10);  

	third_division = (second_division // 10); 
	third_Index = (second_division % 10); 

	fourth_division = (third_division // 10); 
	fourth_index = (third_division % 10); 
		
	fifth_index = fourth_division;
	
	seperated_digit = {"first": fifth_index, "second": fourth_index, "third": third_Index, "fourth": second_index, "fifth": first_index}
	
	return seperated_digit

while(True):
	number = int(input("Enter any five digit: "))
	result = digitseperator(number)
	print(result["first"])
	print(result["second"])
	print(result["third"])
	print(result["fourth"])
	print(result["fifth"])

File: firstcode/firstassignemt/myjavaassignment/DigitSeperator.java
import java.util.Scanner;
	
public class DigitSeperator {
	public static void main(String[] miracle) {
		System.out.println("welcome to my digit seperator");
	
	Scanner input = new Scanner(System.in);
		int number;
	System.out.println("Enter a number");
		number = input.nextInt();
		
	int[] result = digitseperator(number);
	System.out.println(result[0]);
	System.out.println(result[1]);
	System.out.println(result[2]);
	System.out.println(result[3]);
	System.out.println(result[4]);
	
	
		
	
	
}
	public static int[] digitseperator(int number) {
		
		int firstIndex;
		int secondIndex;
		int thirdIndex;
		int fourthIndex;
		int firstDivision;
		int secondDivision;
		int thirdDivision;
		int fourthDivision;
		int fifthIndex;

		firstDivision = (number / 10);  
		firstIndex = (number % 10); 

		secondDivision = (firstDivision / 10); 
		secondIndex = (firstDivision % 10);  

		thirdDivision = (secondDivision / 10); 
		thirdIndex = (secondDivision % 10); 

		fourthDivision = (thirdDivision / 10); 
		fourthIndex = (thirdDivision % 10); 
		
		fifthIndex = fourthDivision;
	
	int[] result = {fifthIndex, fourthIndex, thirdIndex, secondIndex, firstIndex};
	
		return result;
		
}
}